fabric-enchantments-v1/build.gradle

```diff
@@ -0,0 +1,2 @@
+archivesBaseName = "fabric-enchantments-v1"
+version = getSubprojectVersion(project, "v0.1.0")
```

Comment:
No, you still haven't fixed it! Why the *v* prefix? That's not SemVer...

Reply:
Idk why I did this shit.
Finally fixed now.